src/fondant/schemas/common.json

```diff
@@ -7,6 +7,7 @@
         "int8",
         "int16",
         "int32",
+        "int64",
```

Comment:
This can be kept, I would just check that previous component can run at `int32` to reduce memory footprint